learningresources/models.py

```diff
@@ -27,6 +28,12 @@
 STATIC_ASSET_PREFIX = 'assets'
 STATIC_ASSET_BASEPATH = STATIC_ASSET_PREFIX + '/{org}/{course_number}/{run}/'
 
+# Used to signify that a repository has been created
+# and its initial permissions have been set.
+# This allows the "curation status" vocabulary to be
+# created automatically.
+repo_created = Signal(providing_args=["repository"])
```

Comment:
It is explicitly called. I'm just a bobo and forgot I had to move the call to taxonomy signals.

Because:

Originally I added a `post_save` signal for this (before creating the custom `repo_created` signal. However, the taxonomy app _already_ listened for a `post_save` to fix #549. It would be fine to have both as `post_save`, except that they fire in the wrong order. There is no way to set the order. It's actually great that it failed locally or it could have failed randomly in production.

So, I have to call `repo_created` from the permissions signal handler because I have to know the permissions are set before going further (this is actually the original problem that launched #549).

I still lean towards having the handler defined here, since it is regarding the `Repository` model.